src/adapters/PeopleAdapter.js

```diff
@@ -7,10 +7,25 @@ import {throwError} from 'rxjs';
  */
 export default class PeopleAdapter {
   /**
-   * Returns an observable that emits person data
+   * A Person object with details about the person
+   * @typedef {Object} Person
+   * @property {string} id The person identifier
+   * @property {Array.<string>} emails An array of emails for the person
+   * @property {string} displayName The name to be displayed for the person
+   * @property {string} firstName The first name of the person
+   * @property {string} lastName The last name of the person
+   * @property {string} nickName The short name for the person
+   * @property {string} avatar The full url to the person's avatar
+   * @property {string} orgId The id of the organization the person belongs to
+   * @property {string} status The presence status of the user.
+   *                        One of: null, 'active', 'bot', 'call', 'dnd', 'group', 'inactive', 'meeting', 'ooo', 'presenting', 'self', 'typing'
```

Comment:
Do we have a column limit? I really dislike horizontal scrolling. At least on the Github interface this line causes the horizontal scroll

Reply:
Resolved via teams.